ktor-http/common/src/io/ktor/http/Url.kt

```diff
@@ -27,14 +27,15 @@ public class Url internal constructor(
     public val fragment: String,
     public val user: String?,
     public val password: String?,
-    public val trailingQuery: Boolean,
-    private val urlString: String
+    public val trailingQuery: Boolean
```

Comment:
`urlString` should be a separate parameter and it can't be created from the rest, because of encoding/decoding.